front/src/modules/ui/button/components/ButtonGroup.tsx

```diff
@@ -12,17 +12,19 @@ export type ButtonGroupProps = Pick<
   ButtonProps,
   'variant' | 'size' | 'accent'
 > & {
+  style?: React.CSSProperties;
   children: ReactNode[];
 };
 
 export function ButtonGroup({
+  style,
   children,
   variant,
   size,
   accent,
 }: ButtonGroupProps) {
   return (
-    <StyledButtonGroupContainer>
+    <StyledButtonGroupContainer style={style}>
```

Comment:
We already use classname for passing style.

And then `const StyledButtonGroup = styled(ButtonGroup)`

Reply:
@lucasbordeau Actually to be able to customized `ButtonGroup` like that `styled(ButtonGroup)` we need to drill down the style otherwise it will not work